src/ngMock/browserTrigger.js

```diff
@@ -9,16 +9,37 @@
    * This is a global (window) function that is only available when the {@link ngMock} module is
    * included.
    *
-   * It can be used to trigger a native browser event on an element, which is useful for testing.
+   * It can be used to trigger a native browser event on an element, which is useful for unit testing.
    *
    *
    * @param {Object} element Either a wrapped jQuery/jqLite node or a DOMElement
    * @param {string=} eventType Optional event type. If none is specified, the function tries
    *                            to determine the right event type for the element, e.g. `change` for
    *                            `input[text]`.
-   * @param {Object=} eventData An optional object which contains additional event data (such as x,y
-   *                            coordinates, keys, etc...) that are passed into the event when
-   *                            triggered.
+   * @param {Object=} eventData An optional object which contains additional event data that is used
+   *                            when creating the event:
+   *
+   *  - `keys`: an array of possible modifier keys (ctrl, alt, shift, meta) for the event.
+   *
+   *  - `x`: x-coordinates for [MouseEvent](https://developer.mozilla.org/docs/Web/API/MouseEvent)
+   *    and [TouchEvent](https://developer.mozilla.org/docs/Web/API/TouchEvent).
+   *
+   *  - `y`: y-coordinates for [MouseEvent](https://developer.mozilla.org/docs/Web/API/MouseEvent)
+   *    and [TouchEvent](https://developer.mozilla.org/docs/Web/API/TouchEvent).
+   *
+   *  - `relatedTarget`: the related target for
```

Comment:
Maybe also link to [MouseEvent.relatedTarget][1].

[1]: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/relatedTarget